### ISSUE TITLE
Error when creating a tag namespace in OCI

### ISSUE THREAD

**Issue by kolxd** [oracle/terraform-provider-oci#562]
### Terraform and OCI Provider Version

<details>
Terraform v0.11.7                                                                                                                                + provider.oci v2.1.16
</details>

### Description:
When I try to create a oci_identity_tag_namespace I get the following error: 
**oci_identity_tag_namespace.this: value cannot be empty for field TagNamespaceId in path**

The namespace gets created in OCI, but due to the fact that it errors out, I cannot proceed with the creation of oci_identity_tag keys automatically.

### Code used for the resource
```
resource "oci_identity_tag_namespace" "this" {
  compartment_id = "${var.compartment_ocid}"
  description    = "${var.tag_namespace_description}"
  name           = "${var.tag_namespace_name}"
}
```

**Comment by alexng-canuck** [comment]
Hello @kolxd ,

It sounds like the error is showing up when Terraform is trying to refresh the state of the namespace and is not able to find the ID of the created namespace from the state file.

Can you run this command to apply the config with all debug logs enabled?
`DEBUG=true TF_LOG=DEBUG OCI_GO_SDK_DEBUG=1 terraform apply`

Please share the resulting `terraform.tfstate` file and the resulting command output with us.

**Comment by alexng-canuck** [comment]
+ @parweza for FYI.

**Comment by kolxd** [comment]
## Terraform.tfstate
```
{
    "version": 3,
    "terraform_version": "0.11.7",
    "serial": 1,
    "lineage": "57c2b0cf-1dc0-f41a-c525-4caab06454ff",
    "modules": [
        {
            "path": [
                "root"
            ],
            "outputs": {},
            "resources": {},
            "depends_on": []
        }
    ]
}
```
I have ran the command with debug and attached a document with the log:
[Debug_log_taggining.docx](https://github.com/oracle/terraform-provider-oci/files/2248298/Debug_log_taggining.docx)

**Comment by alexng-canuck** [comment]
Hi @kolxd 

We now understand the issue and are working on a fix for it. The fix will be part of the next release of this provider.

Thanks,
Alex

**Comment by alexng-canuck** [comment]
Hi @kolxd ,

This issue should be resolved in the latest v2.1.17 release of the provider. Can you see if this fixes the issue for you?

https://github.com/oracle/terraform-provider-oci/releases

Thanks,
Alex

**Comment by kolxd** [comment]
Hi @alexng-canuck,

The issue is fixed in the latest release. Thank you!

Regards,
Flavius
